test/utils.js

```diff
@@ -36,11 +36,26 @@ export function almostEqual(a, b, criteria) {
   }
 }
 
-export function checkValue(
-    output, expected, criteria = opFp32AccuracyCriteria) {
+function getBitwise(value) {
+  const buffer = new ArrayBuffer(4);
+  const view = new DataView(buffer);
+  view.setFloat32(0, value);
+  return view.getInt32(0);
```

Comment:
should use `getBigInt64` and return a `BigInt` as I commented https://github.com/webmachinelearning/webnn-polyfill/pull/139#discussion_r746319017